src/pages/setup.tsx

```diff
@@ -0,0 +1,92 @@
+/* global window, fetch */
+import React from 'react';
+import { useFormik } from 'formik';
+import { config } from '../api/config';
+
+const SetupPage: React.FC = () => {
+  const formik = useFormik({
+    initialValues: {
+      secret: '',
+      adminSecret: '',
+      discordChannelIds: '',
+      discordBotToken: '',
+      slackBotToken: '',
+      slackSigningSecret: '',
+    },
+    async onSubmit(values) {
+      // TODO: Update all values in the database, and redirect to admin page
+      // QUESTION: Should I bundle the form data into multiple POST requests to the config api, or should I connect this entity directly to the database similarly to the ConfigItem?
```

Comment:
You'll need to make calls to the API, the browser cannot connect to the database